src/client/content_cao.cpp

```diff
@@ -333,13 +333,13 @@ GenericCAO::GenericCAO(Client *client, ClientEnvironment *env):
 	}
 }
 
-bool GenericCAO::getCollisionBox(aabb3f *toset) const
+bool GenericCAO::getCollisionBox(aabb3o *toset) const
 {
 	if (m_prop.physical)
 	{
 		//update collision box
-		toset->MinEdge = m_prop.collisionbox.MinEdge * BS;
-		toset->MaxEdge = m_prop.collisionbox.MaxEdge * BS;
+		toset->MinEdge = v3fToOpos(m_prop.collisionbox.MinEdge) * BS;
```

Comment:
Why are you switching the collision box to `aabb3o` when you're still using floats when reading?

Reply:
readed collision box contains small relative values like -10..10
after this line huge absolute coord added and this should be done with doubles